internal/graph/check.go

```diff
@@ -253,8 +263,9 @@ func intersection(ctx context.Context, concurrencyLimit uint32, handlers ...Chec
 				err = result.err
 				continue
 			}
+			dbReads += result.resp.ResolutionMetadata.DatabaseReads
 
-			if !result.resp.GetAllowed() {
```

Comment:
What's the difference?

Reply:
Previously, `result.resp` was of type `*openfgav1.CheckResponse` which has that method defined in the generated code